src/main/java/org/iitr/muzo/configuration/SecurityConfiguration.java

```diff
@@ -0,0 +1,8 @@
+package org.iitr.muzo.configuration;
+
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
+
+@EnableWebSecurity
+public class SecurityConfiguration extends WebSecurityConfigurerAdapter {
```

Comment:
Why is this for?

Reply:
was trying to add it for Login security config. Couldn't figure it out.